File: Python PowerPoint Examples/SlidesApiDemo/getSlideComments.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "GetSlideComments.pptx"
password = ""
storage = ""
slideIndex = 0
inputFolder = "input"
result = api.get_slide_comments(fileName, slideIndex, password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/SlidesApiDemo/getSlideComments.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "SlideComments.pptx"; 
$password = null; 
$folder = "Powerpoint"; 
$storage = null; 
$slideIndex = 0;
$result = $apiInstance->getSlideComments($name,$slideIndex, $password, $folder, $storage);